src/lib.rs

```diff
@@ -88,11 +88,10 @@ impl Limits {
         }
     }
 
-    // match oopsla limits
     pub fn deriving() -> Self {
         Self {
             iter: 5,
-            node: 100_000,
```

Comment:
I think it's important that we run with the same limits as Oopsla in order for it to be a fair comparison, I don't think we should change this.